### DISCUSSION TITLE
[6주차] [온라인 스터디] 글 모음

### DISCUSSION THREAD

**Discussion by bigsaigon333** [fe-backpackers/fe-interview-basecamp#213] [General]
[6주차] [온라인 스터디] 글 모음

**Comment by bigsaigon333** [comment]
## 마무리 회고

**Reply by ganeodolu** [reply]
오랜만에 다시 인터뷰 스터디에 참여했습니다. 부족한 부분이 여전히 많지만 하나하나 더 이해할 수 있도록 하겠습니다.

**Reply by jinsoobyeon** [reply]
일단 그 전보다 오늘 진행했던 풀어서 설명하는 방식이 더 좋은 것 같아요~
앞으로 어떻게 하면 스터디원에게 더 도움되게 할 수 있는지 다같이 고민이 필요해 보입니다.

**Reply by bigsaigon333** [reply]
## 동동:  스터디 방향을 조금 바꿔보려는 생각을 하고 있습니다. 아이디어 있으시면 말씀 주세요~!

현재 아래 3단계로 스터디가 구성이 되어 있습니다. 
1. 개념 학습
2. 나만의 언어로 정리
3. 온라인 모임에서 피드백 주고받기

### 1. 개념 학습
지금 자유롭게 개념을 선택해서 학습을 하고 있는데, 너무 다양하다보니 서로 얘기하는데 어려움이 있을 때가 있는 것 같아요.
공통주제를 선정하는 건 어떨지 고민됩니다. 


### 2. 나만의 언어로 정리
스터디가 어느 샌가 정리해온 내용을 잘 외우고 있는지에 초점을 맞추고 있다는 생각이 드네요. 
말을 잘 하는 것도 중요하지만, 제대로 이해하는게 더 중요한 것 같습니다.
글 정리에 압박감을 느끼지 않고, 이해를 해나가는 수단으로서 글 정리를 이용했으면 좋겠습니다.


### 3. 온라인 모임에서 피드백 주고받기
단순히 잘 외웠는지 서로 확인해주는 것이 아니라, 모르는 사람에게 설명을 해보는 방식으로 변경합니다. 
파인만 테크닉처럼, 우리 엄마도 알 수 있게끔 쉽게 설명을 해보면서 내가 무엇을 모르는지 셀프 체크하고, 또 상대방으로부터 피드백도 받습니다.
